draft.md

```diff
@@ -37,7 +37,9 @@ Release Date: 2019-00-00
 
 **CRAN**
 
+[{corporaexplorer 0.6.2}: A 'Shiny' App for Exploration of Text Collections](https://github.com/kgjerde/corporaexplorer)
```

Comment:
Thank you! 

Would you mind if we linked to the canonical CRAN page instead?

https://CRAN.R-project.org/package=corporaexplorer

Reply:
Sure, do what is customary for rweekly – thanks!